### PR TITLE
Rename bigtable-dataflow-import to bigtable-dataflow-seqio

### PR DESCRIPTION
**Pull request by igorbernstein2**
Now that we support import & export we should stop calling this project import. Not sure if we should add shims for backwards compatibility in bigtable-dataflow-import

### CODE REVIEW

File: bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ Cloud Dataflow.
 
 ## Instructions
 
-Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-dataflow-import/0.9.5.1/bigtable-dataflow-import-0.9.5.1-shaded.jar), which is an aggregation of all required jars.
+Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-dataflow-seqio/0.9.6/bigtable-dataflow-import-0.9.6-shaded.jar), which is an aggregation of all required jars.
```
Comment (sduskis) [inline bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md]: Let's update the docs in a later PR.  The 0.9.5.1 instructions will still work, while the 0.9.6 will not work yet.

**Comment by sduskis** [comment]
@mbrukman Are you ok with the new package name?  We should hold off committing this until you give your approval.

File: bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ Cloud Dataflow.
...
 
 ## Export
 
 On the command line:
 
 ```
-java -jar bigtable-dataflow-import-0.9.5.1-shaded.jar export \
+java -jar bigtable-dataflow-import-0.9.6-shaded.jar export \
```
Comment (mbrukman) [inline bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md]: The thing I'd like to see changed is the string `bigtable-dataflow-import` in this line and the one below. It looks like this change will not do that?

File: bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Create the table in your cluster.
 On the command line:
 
 ```
-java -jar bigtable-dataflow-import-0.9.5.1-shaded.jar import \
+java -jar bigtable-dataflow-import-0.9.6-shaded.jar import \
```
Comment (mbrukman) [inline bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md]: Would like to see what we're doing with the instance of `bigtable-dataflow-import` here as well.

**Comment by sduskis** [comment]
TSV and CSV files would be the next logical additions.

File: bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md (hunk 1 above)
Comment (igorbernstein2) [inline bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md]: done

**Comment by igorbernstein2** [comment]
after speaking with @mbrukman, the naming convention should be
artifact: bigtable-dataflow-import-export.jar
package: com.google.cloud.bigtable.dataflow.importexport.sequencefile.

I'll create a new PR for that

**Pull request closed without merging**